istioctl/cmd/proxystatus.go

```diff
@@ -35,6 +35,22 @@ import (
 	"istio.io/istio/pkg/kube"
 )
 
+const (
+	// Used for comparison with Istiod configuration when sidecar unable to supply
+	emptyEnvoyDump = `
+{
+		"configs": [
+			{
+				"@type": "type.googleapis.com/envoy.admin.v3.ListenersConfigDump"
+			},
+			{
+				"@type": "type.googleapis.com/envoy.admin.v3.RoutesConfigDump"
+			}
```

Comment:
Do we need clusters? If not can you add a comment why?

Reply:
I removed the empty list of routes and listeners.  I believe you are correct ... it is more clear to fix the logic not to care, than to carefully craft a fake response that works with the existing logic.